python/pyarrow/_orc.pxd

```diff
@@ -54,9 +111,40 @@ cdef extern from "arrow/adapters/orc/adapter.h" \
 
         int64_t NumberOfRows()
 
+        FileVersion GetFileVersion()
+
```

Comment:
Nit: no need to skip lines after every C++ method declaration.

Reply:
Fixed